### DISCUSSION TITLE
Have you tried using Fredy on a Raspberry Pi?

### DISCUSSION THREAD

**Discussion by Giganton0** [orangecoding/fredy#85] [Q&A] [answered]
I was wondering if its suitable for a Raspberry Pi and if so, which version of the raspberry pi is minimum requirement for Fredy to work properly.

**Comment by orangecoding** [comment, marked as answer]
Hey,

Absolutely. Fredy is extremely lightweight. I have it running on a pi 3 with currently 8 jobs in parallel. No issues.
